ban/tests/http/test_schema.py

```diff
@@ -109,3 +110,10 @@ def test_get_position_collection(get, schema):
     factories.PositionFactory()
     resp = get('/position')
     validate_call(resp, schema)
+
+
+@authorize
+def test_get_openapi_sorted(get, schema):
+    source = load(get('/openapi').json)
+    sorted_false = json.dumps(source, sort_keys=False)
+    assert source != sorted_false
```

Comment:
You are comparing a dict to a string :/
Also, as I understand your PR, the initial issue was that the schema had not the correct content-type header, so you may want to test that too.